soap_api/mastr_wind_download.py

```diff
@@ -54,8 +54,6 @@ def setup_power_unit_wind():
     else:
```

Comment:
Theoretically these commits should have gone in a v0.9.1, in their own branch, merged into `dev` then create branch v0.9.1 from `dev` and merge it into master.